examples/widgets/checkbox/lv_example_checkbox_3.c

```diff
@@ -0,0 +1,63 @@
+#include "../../lv_examples.h"
+#if LV_USE_CHECKBOX && LV_BUILD_EXAMPLES
+
+static void event_handler0(lv_event_t * e)
+{
+    lv_event_code_t code = lv_event_get_code(e);
+    lv_obj_t * obj = lv_event_get_target(e);
+    lv_obj_t * other_obj = lv_obj_get_child(lv_obj_get_parent(obj), (1 - lv_obj_get_child_cnt(obj)));
+    if(code == LV_EVENT_VALUE_CHANGED) {
+        /*Handle style changes*/
+        if(lv_obj_get_state(obj) == LV_STATE_CHECKED) {
+            //If other object is also checked, then trigger an event manually for the 3rd object
```

Comment:
```suggestion
            /*If other object is also checked, then trigger an event manually for the 3rd object*/
```
Check other comments too for consistency